sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 - All `DataSource`s now have public properties exposing the associated parameters used to get the data, such as endpoints, connection strings, and query strings.
 
 ### Breaking Changes
+- Moved all `Options` types (e.g., `GetAlertsOptions`) from `Azure.AI.MetricsAdvisor.Models` to the `Azure.AI.MetricsAdvisor` namespace.
+- Moved `TimeMode`, `FeedbackQueryTimeMode`, `GetAnomaliesForDetectionConfigurationFilter`, and `GetDataFeedsFilter` from `Azure.AI.MetricsAdvisor.Models` to the `Azure.AI.MetricsAdvisor` namespace.
```

Comment:
```suggestion
- Moved `AlertQueryTimeMode`, `FeedbackQueryTimeMode`, `GetAnomaliesForDetectionConfigurationFilter`, and `GetDataFeedsFilter` from `Azure.AI.MetricsAdvisor.Models` to the `Azure.AI.MetricsAdvisor` namespace.
```

Reply:
I decided to call it `TimeMode` because that's the name we had in beta 1. We renamed it to `AlertQueryTimeMode` in this beta. Do you think it's better to keep the new name anyway?

Reply:
ohh I missed that. hmm yeah I think it is ok to leave it with previous name. Looking at the full changelog, user can see the rename